README.rst

```diff
@@ -4,7 +4,6 @@ PHP Coding Standards Fixer
 The PHP Coding Standards Fixer tool fixes *most* issues in your code when you
 want to follow the PHP coding standards as defined in the PSR-1 and PSR-2
 documents.
-
 If you are already using ``PHP_CodeSniffer`` to identify coding standards
```

Comment:
If I can as you - could you update it to be more general? PHPCS has it's own CS fixer now, then IMO it is worth to add here more general descr like `if you are using a linter to identify....` or sth